packages/flame_test/lib/src/flame_test.dart

```diff
@@ -30,7 +30,7 @@ extension FlameGameExtension on Component {
 }
 
 typedef GameCreateFunction<T extends Game> = T Function();
-typedef VerifyFunction<T extends Game> = void Function(T);
+typedef VerifyFunction<T extends Game> = dynamic Function(T);
```

Comment:
Note: I don't like dynamics, but here I followed the signature that dart test does, that way we can await for it without compiler errors.

Reply:
I think that is fine in this case, good catch!